flow/scripts/synth_preamble.tcl

```diff
@@ -108,13 +109,30 @@ puts $constr "set_driving_cell $::env(ABC_DRIVER_CELL)"
 puts $constr "set_load $::env(ABC_LOAD_IN_FF)"
 close $constr
 
-proc synthesize_check {report synth_args} {
-  # Generic synthesis
-  log_cmd synth -top $::env(DESIGN_NAME) -run :fine {*}$synth_args
-  json -o $::env(RESULTS_DIR)/$report.json
-  # Run report and check here so as to fail early if this synthesis run is doomed
-  exec -- python3 $::env(SCRIPTS_DIR)/mem_dump.py --max-bits $::env(SYNTH_MEMORY_MAX_BITS) $::env(RESULTS_DIR)/$report.json
-  synth -top $::env(DESIGN_NAME) -run fine: {*}$synth_args
-  # Get rid of indigestibles
-  chformal -remove
+proc convert_liberty_areas {} {
+  cellmatch -derive_luts =A:liberty_cell
+  # find a reference nand2 gate
+  set found_cell ""
+  set found_cell_area ""
+  foreach cell [tee -q -s result.string select -list-mod =*/a:lut=4'b0111 %m] {
```

Comment:
What is this magic about lut?  Is this looking for a nand2 as the message below suggests?  A comment would be helpful for non-yosys experts.

Reply:
Yes, let me add it